src/connectivity.jl

```diff
@@ -748,3 +760,73 @@ function isgraphical(degs::Vector{<:Integer})
     end
     return true
 end
+
+"""
+    isdigraphical(indegree_sequence, outdegree_sequence)
+
+Check whether the given indegree sequence and outdegree sequence are digraphical, that is whether they can be the indegree and outdegree sequence of a digraph.
+
+### Implementation Notes
+According to Fulkerson-Chen-Anstee theorem, a sequence ``\\{(a_1, b_1), ...,(a_n, b_n)\\}`` (sorted in descending order of a) is graphic iff the sum of vertex degrees is even and  ``\\sum_{i = 1}^{n} a_i = \\sum_{i = 1}^{n} b_i\\}`` and the sequence obeys the property -
+```math
+\\sum_{i=1}^{r} a_i \\leq \\sum_{i=r+1}^n min(r-1,b_i) + \\sum_{i=r+1}^n min(r,b_i)
+```
+for each integer r <= n-1. 
```

Comment:
Do you have a source for these conditions? Because the ones one Wikipedia looks differently.

I also notices that this algorithm does return incorrect results in some cases:
```julia
# would only work with a self-loop, but we don't allow these, right?
julia> isdigraphical([1],[1])
true

# would require multi-edges
julia> isdigraphical([5],[5])
true

# but this should also be true if we allow self-loops.
julia> isdigraphical([1,0],[1,0])
false
```

Reply:
Also, I was wondering if latex heavy docstrings are a good idea? We don't need to change it in this PR, as we always can do it latter, the issue is just that in the REPL it looks like this:
```julia
help?> isdigraphical
search: isdigraphical

  isdigraphical(indegree_sequence, outdegree_sequence)

  Check whether the given indegree sequence and outdegree sequence are digraphical, that is whether
  they can be the indegree and outdegree sequence of a digraph.

  Implementation Notes
  ––––––––––––––––––––––

  According to Fulkerson-Chen-Anstee theorem, a sequence \{(a_1, b_1), ...,(a_n, b_n)\} (sorted in
  descending order of a) is graphic iff the sum of vertex degrees is even and \sum_{i = 1}^{n} a_i =
  \sum_{i = 1}^{n} b_i\} and the sequence obeys the property -

  \sum_{i=1}^{r} a_i \leq \sum_{i=r+1}^n min(r-1,b_i) + \sum_{i=r+1}^n min(r,b_i)

  for each integer r <= n-1.

  See also: isgraphical
```